topics/dtk/index.md

```diff
@@ -0,0 +1,11 @@
+---
+aliases: dtk, DTK
+display_name: Data Life Engine
```

Comment:
```suggestion
display_name: Deepin Tool Kit
```

Perhaps this should be Deepin Tool Kit instead of Data Life Engine? Perhaps I'm missing something, though! 😅